core/services/headtracker/head_broadcaster.go

```diff
@@ -54,8 +52,6 @@ type headBroadcaster struct {
 	latest *eth.Head
 }
 
-var _ httypes.HeadTrackable = (*headBroadcaster)(nil)
```

Comment:
Can somebody confirm this is safe to remove? I do not see any init() functions or such that would require this.

Reply:
It's just to ensure that it conforms to the interface, no?

Reply:
If it does not conform, it would not even compile... 
Anyway, after this was removed, no tests was affected, so I would assume this is safe.